lib/main_common.pm

```diff
@@ -2405,6 +2405,24 @@ sub load_security_tests_selinux {
     loadtest "security/selinux/chcat";
 }
 
+sub load_security_tests_cc {
+    # Setup environment for cc testing: "audit-test" test suite setup
+    # Such as: download code branch; install needed packages
+    loadtest "security/cc/cc_audit_test_setup";
+
+    # Run test cases of "audit-test" test suite which do NOT need SELinux env
+    # Such as: set up SELinux with permissive mode and specific policy type
+    # TODO: add more test cases
```

Comment:
Please avoid any "TODO" comments in code as these tend to stay around for very long. Better use an issue tracker to manage tasks

Reply:
Sure.
Done, add a poo to track.